src/util/token.js

```diff
@@ -23,6 +15,27 @@ const token = (seed, publicKey) => {
   )
 }
 
+const authTokenFromSharedSecret = (sharedSecretBuffer) => {
+  return base64url(
+    crypto.createHmac('sha256', sharedSecretBuffer)
+      .update(AUTH_HMAC_INPUT, 'ascii')
+      .digest()
+  )
+}
+
+const authToken = ({ secretKey, peerPublicKey }) => {
+  return authTokenFromSharedSecret(sharedSecret({
+    secretKey, peerPublicKey
+  }))
+}
+
+const token = ({ secretKey, peerPublicKey }) => {
+  // secret and public key should be stored as base64url strings
+  return tokenFromSharedSecret(sharedSecret({
+    secretKey, peerPublicKey
+  }))
+}
```

Comment:
I'm just passing in a map with named fields instead of in-order arguments. It makes it a little harder to mess up parameters by accident.